libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/baseChart/PluggableBaseChart.tsx

```diff
@@ -352,7 +352,20 @@ export class PluggableBaseChart extends AbstractPluggableVisualization {
     ): IChartConfig {
         const { config = {}, customVisualizationConfig = {} } = options;
         const colorMapping: IColorMappingItem[] = get(supportedControls, "colorMapping");
-
+        const { zoomInsight, ...restSupportedControls } = supportedControls;
+        let chart = {};
+        if (zoomInsight) {
+            chart = {
+                zoomType: "x",
+                panKey: "shift",
+                panning: true,
+                resetZoomButton: {
+                    theme: {
+                        display: "none",
+                    },
+                },
+            };
```

Comment:
Please make this constant

Reply:
I have moved it insight the `propertiesHelper` file.